### PR TITLE
docs: add fetch priority

### PR DESCRIPTION
**Pull request by LingyuCoder**
## Summary

- Add documents of fetch priority

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: website/docs/en/api/modules/module-methods.mdx [hunk 1]
```diff
@@ -162,6 +163,14 @@ Tells the browser that the resource might be needed during the current navigatio
 
 A name for the new chunk.
 
+##### webpackFetchPriority
+
+<ApiMeta addedVersion="1.0.0" />
+
+- **Type:**: `"low" | "high" | "auto"`
+
+Set [`fetchPriority`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority) for specific dynamic imports. It's also possible to set a global default value for all dynamic imports by using the `module.parser.javascript.dynamicImportFetchPriority` option.
```
Comment (chenjiahan) [inline website/docs/en/api/modules/module-methods.mdx:172]: I can't find the `module.parser.javascript.dynamicImportFetchPriority` config in the Rspack website, can you add it too

Comment (chenjiahan) [inline website/docs/en/api/modules/module-methods.mdx:172]: Link to the related config is better

```suggestion
Set [`fetchPriority`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority) for specific dynamic imports. It's also possible to set a global default value for all dynamic imports by using the [module.parser.javascript.dynamicImportFetchPriority](/config/module#moduleparserjavascriptdynamicimportfetchpriority) option.
```

File: website/docs/zh/api/modules/module-methods.mdx [hunk 2]
```diff
@@ -160,6 +161,14 @@ import(
 
 指定新 chunk 的名称。
 
+##### webpackFetchPriority
+
+<ApiMeta addedVersion="1.0.0" />
+
+- **类型：**: `"low" | "high" | "auto"`
+
+为指定的动态导入设置 [`fetchPriority`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority)。也可以通过使用 `module.parser.javascript.dynamicImportFetchPriority` 选项为所有动态导入设置全局默认值。
```
Comment (chenjiahan) [inline website/docs/zh/api/modules/module-methods.mdx:170]:
```suggestion
为指定的动态导入设置 [`fetchPriority`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/fetchPriority)。也可以通过使用 [module.parser.javascript.dynamicImportFetchPriority](/config/module#moduleparserjavascriptdynamicimportfetchpriority) 选项为所有动态导入设置全局默认值。
```

File: website/docs/en/api/modules/module-methods.mdx (hunk 1 above)
Comment (LingyuCoder) [reply website/docs/en/api/modules/module-methods.mdx:172]: The module configuration documentation is out-of-date, lost of items are wrong, I will create a new pull request to update it.

Comment (LingyuCoder) [reply website/docs/en/api/modules/module-methods.mdx:172]: do it with module configuration document

File: website/docs/zh/api/modules/module-methods.mdx (hunk 2 above)
Comment (LingyuCoder) [reply website/docs/zh/api/modules/module-methods.mdx:170]: do it with module configuration document

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**